node/src/BaseClient.ts

```diff
@@ -571,6 +571,24 @@ export interface ScriptOptions {
     args?: GlideString[];
 }
 
+/**
+ * Enum of Valkey data types
+ * `STRING`
+ * `LIST`
+ * `SET`
+ * `ZSET`
+ * `HASH`
+ * `STREAM`
+ */
+export enum ObjectType {
```

Comment:
We're quering for types in runtime from glide core in other clients. (lazy cached)
Why not to do the same here?

Reply:
That's not what we did in Python, do you know for a fact that it is better? It doesn't seem like that, we use types for many things and each language has its own impl of enum.

Reply:
In java we query this in runtime from glide-core and this gives us some amount of headache. It could be replaced by hardcoded strings and boost performance of first scan invocation.

Reply:
The goal is to avoid hard-codded data-type?
I don't think it worth it, the chances that core datatype will be added is very low, and if it will happen it is one line per wrapper. It's more pain than gain and more overhead.
Consider the fact that also new data type like geo are sitting under one of those.